package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "opentelemetry-azure-monitor-js",
+  "version": "0.0.0",
+  "description": "Application Insights exporter for the Open Telemetry JavaScript (Node.js) SDK",
+  "main": "build/index.js",
+  "private": true,
+  "scripts": {
+    "build": "tsc -p ./tsconfig.json",
+    "lint": "eslint . --ext .ts,.js",
+    "test": "ts-mocha -p ./tsconfig.json test/**/*.test.ts"
+  },
+  "engines": {
+    "node": ">=8.3.0"
+  },
+  "author": "appinsightssdk@microsoft.com",
+  "license": "MIT",
+  "devDependencies": {
+    "@types/mocha": "^7.0.2",
+    "@types/node": "^13.9.8",
+    "@typescript-eslint/eslint-plugin": "^2.26.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-typescript": "^7.2.0",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.1.2",
+    "husky": "^4.2.3",
+    "lerna": "^3.20.2",
+    "lerna-changelog": "^1.0.1",
+    "mocha": "^7.1.1",
+    "nyc": "^15.0.0",
+    "prettier": "^2.0.2",
+    "ts-mocha": "^7.0.0",
+    "tslint": "^6.1.0",
+    "typescript": "^3.8.3"
+  },
+  "dependencies": {
+    "@opentelemetry/api": "^0.5.2",
+    "@opentelemetry/base": "^0.5.2",
```

Comment:
Just curious. Why are there separate packages for `core` and `base`?

Reply:
I think `base` is meant to be the only pure dependency of vendor exporters. This package is using `core` for misc utils like `NoopLogger` and some high-res time converters.